test/views/templates/returns/PaymentStatusTemplateSpec.scala

```diff
@@ -24,21 +24,27 @@ import views.templates.TemplateBaseSpec
 
 class PaymentStatusTemplateSpec extends TemplateBaseSpec {
 
+  val whatYouOweCalc = "We have worked out what you owe based on any payments you might have made on your account. "
```

Comment:
Could you one new test in this spec, for when an outstanding amount is something different than return total amount (i.e. a partial payment has been made)? Just to prove the content is rendering correctly.

Reply:
Yes thanks for pointing that out.